app/src/main/java/com/qe/qzin/activities/MainActivity.java

```diff
@@ -29,7 +35,7 @@
 
 public class MainActivity extends BaseActivity implements NavigationView.OnNavigationItemSelectedListener{
 
-  ArrayList<Event> events;
+  List<Event> events;
   EventsAdapter eventsAdapter;
   @BindView(R.id.rvEvents) RecyclerView rvEvents;
   private EndlessRecyclerViewScrollListener scrollListener;
```

Comment:
Try to categorize member variables, jumbling all types and mixing is not good practice or not easily readable. Once class grows, will not able to read which vars are for bind, etc